dbt/adapters/athena/column.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+
+from dbt.adapters.base.column import Column
+from dbt.exceptions import DbtRuntimeError
+
+
+@dataclass
+class AthenaColumn(Column):
+    is_iceberg: bool = False
+
+    def is_string(self) -> bool:
+        return self.dtype.lower() in {"varchar", "string"}
+
+    def is_binary(self) -> bool:
+        return self.dtype.lower() in {"binary", "varbinary"}
+
+    def is_timestamp(self) -> bool:
+        return self.dtype.lower() in {"timestamp"}
+
+    @classmethod
+    def string_type(cls, size: int) -> str:
+        return f"varchar({size})" if size > 0 else "varchar"
+
+    @classmethod
+    def binary_type(cls) -> str:
+        return "varbinary"
+
+    @classmethod
+    def timestamp_type(cls, is_iceberg: bool) -> str:
+        # Iceberg does not support timestamp with precision 3, that's why use timestamp(6)
+        return "timestamp(6)" if is_iceberg else "timestamp"
```

Comment:
is it the case also with engine v2? IIRC timestamp(3) was supported in v2.

Reply:
I believe v2 engine will be deprecated soon so that no need to add support for timestamp(3) for iceberg

Reply:
fair enough.